### PR TITLE
Create WND_Setup.adoc

### PR DESCRIPTION
**Pull request by voroyam**
here is a how to make WND work in the appliance.

### CODE REVIEW

**Comment by mmattel** [comment]
@voroyam Thanks for adding that !

If there is a reference to an internal docs page like that one you refrenced to:
https://doc.owncloud.org/server/10.0/admin_manual/enterprise/external_storage/windows-network-drive_configuration.html#wnd-listen[ our documentation]

Please always use `xref:` to an .adoc page and not a link to http(s)
See the `best practices` guide in the readme section we created for more help 😄 

It would be in this case:
`xref:enterprise/external_storage/windows-network-drive_configuration.adoc#wnd-listen[our documentation]`

**Comment by mmattel** [comment]
@patrickjahns PR is updated and your comment has been added, pls re-check

File: modules/administration_manual/pages/appliance/wnd_setup.adoc [hunk 1]
```diff
@@ -0,0 +1,65 @@
+= Windows Network Drive Configuration
```
Comment (settermjd) [inline modules/administration_manual/pages/appliance/wnd_setup.adoc]: As you're using WND in the first paragraph, please add it as (WND) at the end of the title.

File: modules/administration_manual/pages/appliance/wnd_setup.adoc [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+systemctl enable owncloud-wnd-listen-SERVER-SHARE.service
+systemctl start  owncloud-wnd-listen-SERVER-SHARE.service
+....
+
+== WND Process Queue
+
+Create or add a `crontab` file in `/etc/cron.d/oc-wnd-process-queue`.
+
+* Make a `crontab` entry to run a script iterating over all `SERVER SHARE` pairs with
+an appropriate `occ wnd:process-queue` command. The commands must be **strictly sequentially**.
```
Comment (settermjd) [inline modules/administration_manual/pages/appliance/wnd_setup.adoc]: "strictly sequentially" should be "strictly sequential".

File: modules/administration_manual/pages/appliance/wnd_setup.adoc [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+....
+* * * * *     root  /usr/bin/univention-app shell owncloud occ wnd:process-queue -vvv SERVER SHARE
+....
+
+== Further Reading
+
+Please see also:
+
+https://central.owncloud.org/t/wnd-listener-configuration/3114[The ownCloud forum] and the
```
Comment (settermjd) [inline modules/administration_manual/pages/appliance/wnd_setup.adoc]: Please convert this to a bulleted list, as additional links may be added in the future.

**Pull request merged**